### DISCUSSION TITLE
Paper background image

### DISCUSSION THREAD

**Discussion by xxxHunterxxx19** [clientIO/joint#2647] [Q&A]
### Introduction

how can I open SVG,PNG, JPEG files and display them on Paper?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment]
There are different ways to display an image in a JointJS diagram.

The most straightforward option is to use the `<image/>` SVG tag: https://developer.mozilla.org/en-US/docs/Web/SVG/Element/image

This example shows how to integrate images that are SVGs specifically: https://www.jointjs.com/demos/external-svg-images

Here's a few more example with images: https://www.jointjs.com/demos?feature=External+images

**Reply by xxxHunterxxx19** [reply]
That's not the answer. I need to open the image and stretch it over the entire Paper. There is not even close to this in any example

**Reply by kumilingus** [reply]
Next time, try to phrase your question better so that everyone understands it and doesn't have to guess.

You need to set a [paper background](https://resources.jointjs.com/docs/jointjs/v4.0/joint.html#dia.Paper.prototype.options.background).

Here are some examples:
https://www.jointjs.com/demos/paper-attributes
https://www.jointjs.com/demos/transitions
